src/templates/en/2019/chapter.html

```diff
@@ -112,7 +112,7 @@ <h2 class="header">Index</h2>
             <h1 class="title">
                 {{ metadata.get('title') }}
             </h1>
-            <img src="/static/images/2019/{{ get_chapter_image_dir(metadata) }}/hero_lg.jpg" class="content-banner">
+            <img loading="lazy" src="/static/images/2019/{{ get_chapter_image_dir(metadata) }}/hero_lg.jpg" class="content-banner">
```

Comment:
Main image should not be lazy-loaded IMHO

```suggestion
            <img src="/static/images/2019/{{ get_chapter_image_dir(metadata) }}/hero_lg.jpg" class="content-banner">
```